lib/handlers/app/forms.js

```diff
@@ -19,6 +20,7 @@ router.use(fhmbaasMiddleware.envMongoDb.getEnvironmentDatabase);
 
 router.use(fhForms.middleware.parseMongoConnectionOptions);
 
+router.use(formsMiddleware.fixFormDbUrl);
```

Comment:
Is the intention here to just fix any app connections strings on the fly at request time?

Reply:
Yes, for forms requests only.